### PR TITLE
Memory Page

### PR DESCRIPTION
**Pull request by ERICCYS**
Able to fetch the latest 10 training information and display on the memory page

### CODE REVIEW

File: client/app.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "pages":[
-    "pages/login/login",
```
Comment (MaxPoon) [inline client/app.json]: We don't want the login page?

File: client/pages/memories/memories.js [hunk 2]
```diff
@@ -26,7 +28,21 @@ Page({
    * 生命周期函数--监听页面显示
    */
   onShow: function () {
-
+    var that = this;
+    qcloud.request({
+      url: 'http://localhost:5757/weapp/memeory',
```
Comment (MaxPoon) [inline client/pages/memories/memories.js]: Use the url from config

File: server/controllers/memory.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const assert = require('assert');
+const { mysql } = require('../qcloud.js');
+const trainingTableName = "Training";
+
+async function get(ctx) {
+  var memeory = await mysql(trainingTableName).orderBy('UpdatedTime', 'desc').select('*').limit(10);
```
Comment (MaxPoon) [inline server/controllers/memory.js]: Yes, setting a limit is good. In the future we can do pagination.

File: client/pages/memories/memories.js [hunk 4]
```diff
@@ -26,7 +28,21 @@ Page({
...
+      method: 'GET',
+      header: {
+        'content-type': 'application/x-www-form-urlencoded'
+      },
+      success: function (res) {
+        var data = res.data;
+        that.setData({
+          trainings: data
+        })
+        console.log(that.data.trainings);
```
Comment (MaxPoon) [inline client/pages/memories/memories.js]: Remove it

File: server/controllers/memory.js (hunk 3 above)
Comment (ERICCYS) [reply server/controllers/memory.js]: Good suggestion

File: client/pages/memories/memories.js (hunk 4 above)
Comment (ERICCYS) [reply client/pages/memories/memories.js]: Okay

File: client/pages/memories/memories.js (hunk 2 above)
Comment (ERICCYS) [reply client/pages/memories/memories.js]: Okay

**Review by MaxPoon — APPROVED** [review]

**Pull request merged**